### ISSUE TITLE
Existing remote dcrd certificate unsupported curve

### ISSUE THREAD

**Issue by davecgh** [decred/decrediton#2855]
Virtually all existing dcrd setups use certificates with the P-521 curve.  Unfortunately, it appears that v1.6.0 of Decrediton no longer supports that curve (probably because Chromium which electron uses removed support for it).

The consequence is that all existing setups that have Decrediton configured to connect to the remote dcrd instance now fail as follows:

![image](https://user-images.githubusercontent.com/2115102/98059654-3f47a480-1e0d-11eb-89bf-7099957c5a2a.png)

I **highly** suggest Decrediton detect the error and give the users clear instructions on how to resolve it  I've put up a PR to dcrd to change the default curve to P-256, but that obviously won't fix all of the existing certs and setups.  The way the user will need to fix it is to remove the `rpc.cert` and `rpc.key` and restart dcrd with the `--tlscurve=P-256` param (or that will be the default curve once the aforementioned PR lands) to allow it to generate a cert and key with that supported curve.

Ideally, I think Decrediton itself should just examine the certificate the user specifies to see if it has a supported curve, and if not, provide a clear message why the cert can't be used and how to resolve it.  Barring that, it should at least detect the error when trying to connect to dcrd.

**Comment by bgptr** [comment]
on this
